### ISSUE TITLE
[Bug]: the returned inserted id are identical. (Restful api)

### ISSUE THREAD

**Issue by shanghaikid** [milvus-io/milvus#27856]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

I inserted 10 items, but the returned insertIds are identical

```
 insert {
        code: 200,
        data: {
          insertCount: 10,
          insertIds: [
            445127262333941300,
            445127262333941300,
            445127262333941300,
            445127262333941300,
            445127262333941300,
            445127262333941300,
            445127262333941300,
            445127262333941300,
            445127262333941300,
            445127262333941300
          ]
        }
      }
```

### Expected Behavior

As we know that the javascript engine(v8) doesn't support `Int64` natively, so if we have int64 in the response, when serializing,  it will be rounded by the js engine.  and there is no way to fix this on the client side, because this is just an API, not the sdk.  

I think milvus should return `string` other than `int64` in the response.

**Comment by shanghaikid** [comment]
and this should be fixed as soon as possible, because most vector operations need `id`, eg: `get`, `delete` operations, the server should handle not only  the `int64` id in the response, but also the `string` id in the request.

**Comment by yanliang567** [comment]
@shanghaikid please help to verify the fix
/assign @shanghaikid

**Comment by shanghaikid** [comment]
it was fixed in v2.3.3
